lib/src/polyline.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:math' as math;
+
+import 'package:turf/helpers.dart';
+
+class Polyline {
+  static String _encode(current, previous, factor) {
```

Comment:
I'm always a fan of declaring types, but for private functions it's not so important. You decide, if you want to give a little more type info